docs/de/firefox_marketing.md

```diff
@@ -0,0 +1,220 @@
+* Firefox Quantum (Desktop-Browser)
+* Firefox Mobile (für iOS und Android)
+* [Firefox Klar](https://play.google.com/store/apps/details?id=org.mozilla.klar&referrer=adjust_reftag%3DcysPvmkarFCOT%26utm_source%3DProduct%2BMarketing%2B%2528not%2Bads%2529%26utm_campaign%3Dffy_klar)
+* [Firefox Monitor](https://monitor.firefox.com/)
+* [Pocket](https://play.google.com/store/apps/details?id=com.ideashower.readitlater.pro&hl=de)
+* [Firefox Reality](https://mixedreality.mozilla.org/firefox-reality/) (VR Browser)
+
+## Persönlichkeit & Tonalität
+
+### Mit wem sprechen wir?
+
+Wir beschreiben unsere Zielgruppe als Conscious Choosers. Damit sind Menschen gemeint, die bewusste Entscheidungen treffen oder treffen wollen – online und offline.
+
+Conscious Choosers:
+* entwickeln Vorlieben und Konsumentscheidungen sorgsam und bewusst
+* streben danach, Handlungen mit ihren Werten und Idealen in Einklang zu bringen
+* sind anspruchsvolle Kunden
+* Recherchieren oft selbst, um alle Optionen zu kennen und zu verstehen
+
+### Unsere Markeneigenschaften/Firefox Brand Personality
+
+|Eigenschaften |Persönlichkeit |
+| -------| ---------|
+|opinionated/meinungsstark | **Firefox handelt aus Überzeugung und nimmt selbstbewusst klare Positionen ein.** <br>Unsere Produkte zeigen, dass sich unsere Motivation aus unserer Brand-Vision ableitet. Unser Brand-Ethos liegt allem, was wir tun, zugrunde. Und das wollen wir auch an unsere Nutzern und Partner kommunizieren.</br> |
+|open/offen |**Firefox findet, dass das Internet für alle zugänglich und sicher sein sollte und strebt immer ein offenes Gespräch oder eine Zusammenarbeit an.** <br>Open-minded. Open-hearted. Open source. Eine globale Perspektive ist fester Bestandteil unserer Marke. Wir sprechen viele Sprachen und bemühen uns darum verschiedene Blickwinkel einzunehmen.</br>|
+|kind/einfühlsam|**Firefox antizipiert Bedürfnisse und bietet dort Lösungen und Alternativen an, wo Freiheit und Rechte der User bedroht sind oder missbraucht werden können.** <br>Wir wollen das Beste für unsere Nutzer und die Welt. Also gehen wir mit gutem Beispiel voraus: wir entwickeln bessere Produkte, wir starten Dialoge, wir arbeiten offen und mit anderen zusammen, wir bilden uns und andere weiter und wir informieren unsere Nutzer. Dabei handeln wir allen Menschen gegenüber empathisch.</br>|
+|radical/radikal|**Firefox stellt den Status quo und gängige Praktiken der Tech-Giganten infrage und kämpft mit Zuversicht für ein besseres Internet.** <br>Optimistisch in die Zukunft des Internets zu schauen, ist ein radikaler Akt. Die Bedürfnisse der Nutzer vor unsere eigenen zu stellen ist ebenso radikal. Wir rütteln am Status quo, weil wir finden, dass es richtig ist.</br>|
+
+Unsere Marken- Persönlichkeit wird von dem Zusammenspiel dieser Eigenschaften definiert. Je nach Kontext kann eine Eigenschaft deutlicher oder weniger deutlich hervorstechen.
+
+![Firefox Personality Matrix](/images/firefox_marketing/firefox_personality-de.png)
```

Comment:
Your file is called `images/firefox_marketing/firefox_personality-de.png-de.png` (check the bottom of the PR.

You should fix the file name and call it `firefox_personaly_de.png`, also avoid mixing dash and underscore